Add MSI module management and user guide link (#3440)
magento/devdocs: guides/v2.3/comp-mgr/install-extensions/inventory-management-installation.md

---
group: software-update-guide
title: Manage Inventory Management modules
---

Inventory Management modules provide all inventory features and options for Single and Multi Source merchants to manage product quantities and stock for sales channels. These features are available in Magento 2.3.X Open Source and Commerce.

These features and extensions were developed as part of the [Multi Source Inventory (MSI) project](https://github.com/magento-engcom/msi) through the Magento Community Engineering program.

> ## Installation

Reviewer: The other headings start with verbs, so maybe change this to "Install Inventory Management".

> Inventory Management installs like a Core Bundled Extension (CBE) with all features enabled by default. No additional steps are required for installing and enabling these inventory features. 

Reviewer: I'm not sure the CBE distinction is all that important to external users. I suggest saying something like

Inventory Management installs with Magento Open Source with all features enabled by default.

Author: Sounds good! Edited

        'Magento_InventoryConfiguration' => 1,
        'Magento_InventoryConfigurationApi' => 1,
        'Magento_InventoryGroupedProduct' => 1,
        'Magento_InventoryGroupedProductAdminUi' => 1,
        'Magento_InventoryGroupedProductIndexer' => 1,
        'Magento_InventoryImportExport' => 1,
        'Magento_InventoryIndexer' => 1,
        'Magento_InventoryLowQuantityNotification' => 1,
        'Magento_InventoryLowQuantityNotificationAdminUi' => 1,
        'Magento_InventoryLowQuantityNotificationApi' => 1,
        'Magento_InventoryMultiDimensionalIndexerApi' => 1,
        'Magento_InventoryProductAlert' => 1,
        'Magento_InventoryReservations' => 1,
        'Magento_InventoryReservationsApi' => 1,
        'Magento_InventoryCache' => 1,
        'Magento_InventorySalesAdminUi' => 1,
        'Magento_InventorySalesApi' => 1,
        'Magento_InventorySalesFrontendUi' => 1,
        'Magento_InventoryShipping' => 1,
        'Magento_InventorySourceDeductionApi' => 1,
        'Magento_InventorySourceSelection' => 1,
        'Magento_InventorySourceSelectionApi' => 1,
        'Magento_InventoryShippingAdminUi' => 1,
```

> ## Enable features

Reviewer: ```suggestion
## Enable Inventory Management features
```

> All Inventory Management features enable by default. The **Manage Stock** option in the Admin enables and disables tracking of inventory quantities, calculations of aggregated salable quantities, and reservation management for tracking purchases through to invoice and shipment. 

Reviewer: I think you probably need to rewrite this sentence so that "tracking" isn't used twice. The construction is a bit stilted, and I don't know how to fix it.

Author: Totally revised to shift the focus!

For more information, see [Enabling Inventory Management](https://docs.magento.com/m2/ce/user_guide/catalog/inventory.html) in the Magento 2 User Guide.

## Disable Inventory Management

You may need to disable Inventory Management modules:

> * To speed up upgrade of merchants currently on Magento 2.0/2.1/2.2 and migrating to 2.3.0.

Reviewer: ```suggestion
* To speed up the upgrade process for merchants currently on Magento 2.0/2.1/2.2 and migrating to 2.3.0.
```

> * To support inventory customization. For example, if you have a custom implementation of Multi-Sourcing or Magento Order Management merchants.

Reviewer: MOM merchants doesn't right, especially since you call it out in the next step.

Perhaps say something like 

* To keep using custom or third-party modules that implement Multi-Sourcing.

Author: Revised all the bullets. thanks!

* To use [Magento Order Management](https://omsdocs.magento.com). The current Order Management connector does not support MSI interfaces. We plan to support Inventory Management and Order Management through a connector in a later release.

> To disable, see the instructions for [Enable or disable modules]({{ page.baseurl }}/install-gde/install/cli/install-cli-subcommands-enable.html). When complete, you should see the following modules and values in `composer.json`:

Reviewer: ```suggestion
To disable Inventory Management, see the instructions for [Enable or disable modules]({{ page.baseurl }}/install-gde/install/cli/install-cli-subcommands-enable.html). When complete, you should see the following modules and values in `composer.json`:
```
